pkg/cmd/cli/cmd/request_project.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	projectapi "github.com/openshift/origin/pkg/project/api"
+)
+
+type NewProjectOptions struct {
+	ProjectName string
+	DisplayName string
+	Description string
+
+	Client client.Interface
+}
+
+func NewCmdRequestProject(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	options := &NewProjectOptions{}
+
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("%s <project-name> [--display-name=<your display name> --description=<your description]", name),
+		Short: "request a new project",
+		Long:  `Provisions a new project with you as the project admin.  After completion, switch to your project using: osc project <project-name>.`,
```

Comment:
I meant more like `newapp` or `edit` or `project`... you know, some meat on it. :)